airflow/contrib/kubernetes/pod_launcher.py

```diff
@@ -117,6 +135,35 @@ def read_pod(self, pod):
                 'There was an error reading the kubernetes API: {}'.format(e)
             )
 
+    def _extract_xcom(self, pod):
+        resp = stream(self._client.connect_get_namespaced_pod_exec,
```

Comment:
you should rename stream to something like `kubernetes_stream` since stream is pretty ambiguous.